packages/cra-template/template/.add-ons/bootstrap/index.scss

```diff
@@ -0,0 +1,45 @@
+// Configuration
+@import 'bootstrap/scss/functions';
+@import 'bootstrap/scss/variables';
+@import 'bootstrap/scss/mixins';
+@import 'bootstrap/scss/utilities';
+
+// Base imports
+@import 'bootstrap/scss/root';
+@import 'bootstrap/scss/reboot';
+@import 'bootstrap/scss/type';
+
+// Layout & components
+@import 'bootstrap/scss/badge';
+@import 'bootstrap/scss/card';
+@import 'bootstrap/scss/close';
+@import 'bootstrap/scss/containers';
+@import 'bootstrap/scss/grid';
+@import 'bootstrap/scss/buttons';
+@import 'bootstrap/scss/forms';
```

Comment:
Should we initially comment on these so that later when developers work on the app they can remove the comment by themself to use only components they needed 💡

Reply:
Nice catch! 
Done in 43227fe :)